.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+
+Before submitting a pull request, please verify the following:
+
+* [ ] If you've added code that should be tested, please add tests.
+* [ ] If you've modified the API (e.g. added a new config or public method), update the [docs](https://github.com/getsentry/raven-js/tree/master/docs) and TypeScript [declaration file](/getsentry/raven-js/blob/master/typescript/raven.d.ts).
+* [ ] Ensure your code lints and the test suite passes (npm test).
```

Comment:
I'd prefer people verify the tests locally before pushing a PR. Often people push PRs, see Travis fails, then go "oh well" and never resolve the issues – ends up being a waste of my time just to look at it.